code/game/gamemodes/changeling/powers/superhuman_physiology.dm

```diff
@@ -0,0 +1,16 @@
+/datum/action/changeling/superhuman
+	name = "Superhuman Physiology"
+	helptext = "We may toggle our superhuman biology on and off to avoid detection."
```

Comment:
```suggestion
	helptext = "We can toggle our superhuman biology on and off to avoid detection."
```

Reply:
wait.... thats not even something you can do. description is just wrong here

Reply:
Fixed